streaming/stream_transfer_task.cc

```diff
@@ -230,15 +230,6 @@ future<> stream_transfer_task::execute() {
             }
             return send_mutation_fragments(std::move(si));
         });
-    }).then([this, plan_id, cf_id, id] {
-        sslog.debug("[Stream #{}] SEND STREAM_MUTATION_DONE to {}, cf_id={}", plan_id, id, cf_id);
-        return session->ms().send_stream_mutation_done(id, plan_id, _ranges,
```

Comment:
Don't we still need to make this call, so that receive_task_completed() is called on the other node?

Reply:
I think we do, and I remember considering it. Maybe I only considered the server side and missed the client side.

Reply:
btw, @asias please review too

Reply:
We still need to send STREAM_MUTATION_DONE on the client side.

Reply:
Otherwise looks good to me.